src/vs/workbench/services/editor/browser/editorService.ts

```diff
@@ -42,7 +42,7 @@ import { IModelService } from 'vs/editor/common/services/modelService';
 import { ILogService } from 'vs/platform/log/common/log';
 
 type CachedEditorInput = ResourceEditorInput | IFileEditorInput | UntitledTextEditorInput;
-type OpenInEditorGroup = IEditorGroup | GroupIdentifier | SIDE_GROUP_TYPE | ACTIVE_GROUP_TYPE;
+type OpenInEditorGroup = IEditorGroup | GroupIdentifier | SIDE_GROUP_TYPE | ACTIVE_GROUP_TYPE | NEW_GROUP_TYPE;
```

Comment:
I find `NEW_GROUP_TYPE` is not fully expressing what it does. Based on the implementation it seems to create a new group after the last group by appearance. If so I think the name should reflect that, because as a user of the API one might think that it opens a new group from the currently active group and not the last group.

Reply:
I will remove this with new approach.